### DISCUSSION TITLE
ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(PostgresError { code: "42P05", message: "prepared statement \"s0\" already exists", severity: "ERROR", detail: None, column: None, hint: None }), transient: false })]

### DISCUSSION THREAD

**Discussion by diyoifa** [prisma/prisma#24167] [Q&A]
### Bug description

ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(PostgresError { code: "42P05", message: "prepared statement \"s0\" already exists", severity: "ERROR", detail: None, column: None, hint: None }), transient: false })] 

  15 async migrateClient(client: FirebaseFirestore.DocumentData){
  16     try {
  17         const phoneNumber = client.phoneNumber.includes('+') ? parseInt(client.phoneNumber.replace('+', ''), 10) : parseInt(client.phoneNumber);   
→ 18         const clientExists = await this.prisma.userInfo.findUnique(
Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(PostgresError { code: "42P05", message: "prepared statement \"s0\" already exists", severity: "ERROR", detail: None, column: None, hint: None }), transient: false })] Error checking if client exists

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```
    "@prisma/client": "^5.12.1",

```

**Comment by ludralph** [comment]
Hi @diyoifa 👋 

Thank you for raising this question.

1. What is the name of the database you are using?
2. Is it using a pooled connection to the database? 
3. Are you using Pgbouncer?
4. When do you get this error?

**Reply by diyoifa** [reply]
hi bro I already solved it I was using supabase I update my env variables to this and it worked # Direct connection to the database. Used for migrations.
DIRECT_URL="postgres://postgres.[YOUR-PROJECT-ID]:[YOUR-PASSWORD]@aws-0-us-west-1.pooler.supabase.com:5432/postgres"    

# PostgreSQL connection string with Supavisor config — used by Prisma Client
DATABASE_URL="postgres://postgres.[YOUR-PROJECT-ID]:[YOUR-PASSWORD]@aws-0-[REGION].pooler.supabase.com:6543/postgres?pgbouncer=true&connection_limit=1"   it seems like i was creating many conections to my supabase so I limited to 1

**Reply by bandhan-majumder** [reply]
@diyoifa Thanks. Helped!
